config/routes.rb

```diff
@@ -44,6 +44,8 @@
           put "/contact", on: :member, to: "contacts#update"
           get "/visas", on: :member, to: "visas#edit"
           put "/visas", on: :member, to: "visas#update"
+          get "/users", on: :member, to: "users#index"
+          get "/request-access", on: :member, to: "access_requests#new"
```

Comment:
These routes sit outside the recruitment cycle namespace. The path to navigate to the index page should be, eg—
`/publish/organisations/2GU/users`. This will help keep the redirect from Old Publish simple by matching the same structure that's used there.

Reply:
good spot, commited 👍